core/app/models/spree/stock/content_item.rb

```diff
@@ -43,6 +43,10 @@ def quantity
         # but this massively simplifies things for now
         1
       end
+
+      def volume
+        variant.volume * quantity
```

Comment:
It would be best to add a spec for this method so that it isn't accidentally removed.